### DISCUSSION TITLE
Finish 2024-03 and Start 2024-06

### DISCUSSION THREAD

**Discussion by merks** [eclipse-simrel/simrel.build#288] [Announcements]
The 2024-03 release is final:

- https://download.eclipse.org/releases/2024-03
- https://download.eclipse.org/releases/latest

Download are available:

- https://www.eclipse.org/downloads/packages

---

The 2024-06 release is started:

- https://download.eclipse.org/releases/2024-06
- https://download.eclipse.org/releases/milestone

The 2024-06 staging repository exists:

- https://download.eclipse.org/staging/2024-06

---

Orbit M0 milestones repositories for 2024-06 / 4.32 have been created.

Aggregation - All of below

- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2024-06
- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/milestone/latest

BND - Wrapped

- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/2024-06
- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/nightly/latest

OSGi - Direct from Maven

- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/2024-06
- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/milestone/latest

Native Signed

- https://download.eclipse.org/tools/orbit/simrel/maven-sign/2024-06
- https://download.eclipse.org/tools/orbit/simrel/maven-sign/milestone/latest

Legacy - Build from Source

- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/2024-06
- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/milestone/latest

**Comment by pcdavid** [comment]
@merks Would this be the right place to recall the important dates (milestones, RC, GA) for 2024-06?

I never know where to find the info except in the shared "Eclipse Planning Council / Eclipse Simrel Release Train Calendar", which I don't even recall how I subscribed to many many years ago.

**Reply by merks** [reply]
I have this in pinned tab in my browser:

https://calendar.google.com/calendar/u/0/embed?src=gchs7nm4nvpm837469ddj9tjlk@group.calendar.google.com&dates=20231201/20240331&hl=en&mode=AGENDA

The wiki content got migrated to here:

https://github.com/eclipse-simrel/.github/blob/main/wiki/Simultaneous_Release.md

So there should always be a page like this:

https://github.com/eclipse-simrel/.github/blob/main/wiki/SimRel/2024-06.md

So I think you are suggesting a link to that page in the "announcement" discussion would be a good thing, right?

**Reply by pcdavid** [reply]
> https://github.com/eclipse-simrel/.github/blob/main/wiki/SimRel/2024-06.md

That's exactly what I was looking for, thanks.

> So I think you are suggesting a link to that page in the "announcement" discussion would be a good thing, right?

IMO yes. The thing is, that page exists, but I don't know how I could have discovered it by myself starting from https://github.com/eclipse-simrel. Maybe the [org-global README](https://github.com/eclipse-simrel/.github/tree/main/profile/README.md) could have a link to [the wiki](https://github.com/eclipse-simrel/.github/tree/main/wiki)?

**Reply by merks** [reply]
I've added more helpful information on the organization page.  Thanks for the good suggestions!

**Comment by Phillipus** [comment]
> The 2024-06 release is started:
>    - https://download.eclipse.org/releases/2024-03

Should that be https://download.eclipse.org/releases/2024-06 ?

**Reply by merks** [reply]
Yes.  Fixed.  Thanks.

**Reply by merks** [reply]
Yes. Fixed. Thanks.
